packages/noya-designsystem/src/components/FillInputField.tsx

```diff
@@ -39,6 +40,13 @@ export default forwardRef(function FillInputField(
   const [state] = useApplicationState();
   const { inputBackground, placeholderDots } = useTheme().colors;
 
+  const backgroundPattern = useObjectURL(
```

Comment:
We might need to move this call into a wrapper component, since we don't really want to call it for things that aren't patterns

Reply:
I just pushed a fix for this - I made the background fill a separate component to solve this issue